### DISCUSSION TITLE
vue 多入口怎么使用相同的pinia  我已经确保所有的入口文件都导入相同的pinia实例  但是在项目a中改变了pinia数据 项目 b中不能访问到

### DISCUSSION THREAD

**Discussion by baijuyibuyiju** [vuejs/pinia#2288] [Help and Questions]
vue 多入口怎么使用相同的pinia  我已经确保所有的入口文件都导入相同的pinia实例  但是在项目a中改变了pinia数据 项目 b中不能访问到

**Comment by baijuyibuyiju** [comment]
![1689835469848](https://github.com/vuejs/pinia/assets/71108831/f8208bd2-2f4e-498c-ab5d-215fd7ede64c)
![1689835503115](https://github.com/vuejs/pinia/assets/71108831/69d7f37b-8187-4901-9a2e-cd38123aa3fa)
![1689835518871](https://github.com/vuejs/pinia/assets/71108831/a9040544-2d06-4455-8921-46e14ffac5ae)

![(SNZ6(BQ(P2MS ICLDV)MWM](https://github.com/vuejs/pinia/assets/71108831/bc2cb2cc-b637-4138-9a39-5988b395b744)

**Reply by rush-all** [reply]
一个项目为什么会有多个入口文件
